### PR TITLE
MessageList HTML starter code

### PR DESCRIPTION
**Pull request by sarahsaturday**

### CODE REVIEW

File: scripts/message/PrivateMessages.js
Comment (chesney-hardin) [inline scripts/message/PrivateMessages.js:1]: 1. make sure you are checking that the message.recipientId===currentUser.userId 
-so you also need to import getCurrentUser from TransientState.js

2. i don't think you need the return in the .filter function? 
- make a new variable for the new array of filtered messages and then iterate over that array with .map

**Review by chesney-hardin — CHANGES_REQUESTED** [review]

File: scripts/message/PrivateMessages.js
Comment (sarahsaturday) [reply scripts/message/PrivateMessages.js:1]: gpt says:

The return statement in the .filter function is necessary to indicate whether a message should be included in the filtered array. In this case, we want to return true when the condition is met (message.recipientId === currentUser.userId), indicating that the message should be included.

**Review by chesney-hardin — CHANGES_REQUESTED** [review]

**Review by chesney-hardin — APPROVED** [review]
yay!

**Review by Djmyers1991 — APPROVED** [review]
I approve you.

**Pull request merged**